{{cookiecutter.project_shortname}}/{{cookiecutter.project_shortname}}/__init__.py

```diff
@@ -17,7 +17,7 @@
     _sys.exit(1)
 
 _basepath = _os.path.dirname(__file__)
-_filepath = _os.path.abspath(_os.path.join(_basepath, 'package.json'))
+_filepath = _os.path.abspath(_os.path.join(_basepath, '..','package.json'))
```

Comment:
Hmm will this actually work once the package is installed via `pip`?

Reply:
Maybe this is why that `copyfile` was there?

Reply:
I'll add a test that will package the build and run an app with it installed as a tarball.